CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+- [FIX] Enabled `reduce` and other reduce related parameters to be set when using
+  `MultipleRequestBuilder`.
```

Comment:
An implementer would have to add the new methods.

Updated to `[NEW]` in e6f6be9.